tests/commands/test-kob-version.sh

```diff
@@ -1 +1,92 @@
 
+#!/bin/bash
+
+status="true"
+
+function __test_kob_init
+{
+    echo "Checking for kob..."
+    if [[ -d $KOBMAN_DIR ]]; then
+        source $KOBMAN_DIR/src/kobman-utils.sh
+        __kobman_echo_no_colour "kob found"
+        source $KOBMAN_DIR/bin/kobman-init.sh
+    else
+        status="false"
```

Comment:
remove unwanted echo , only need it to check if kob is present , if not install test should fail and exit with the reason.